villas/controller/components/managers/kubernetes_simple.py

```diff
@@ -0,0 +1,82 @@
+from villas.controller.components.managers.kubernetes import KubernetesManager
+from villas.controller.components.simulators.kubernetes import KubernetesJob
+
+parameters_simple = {
+    'type': 'kubernetes',
+    'category': 'simulator',
+    'uuid': None,
+    'name': '',
+    'properties': {
+        'job': {
+            'apiVersion': 'batch/v1',
+            'kind': 'Job',
+            'metadata': {
+                'name': ''
+            },
+            'spec': {
+                'activeDeadlineSeconds': 3600,
+                'backoffLimit': 2,
+                'template': {
+                    'spec': {
+                        'restartPolicy': 'Never',
+                        'containers': [
+                            {
+                                'image': '',
+                                'imagePullPolicy': 'Always',
+                                'name': 'jobcontainer',
+                                'securityContext': {
+                                    'privileged': False
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        }
+    }
+}
+
+
+class KubernetesManagerSimple(KubernetesManager):
+
+    def __init__(self, **args):
+        super().__init__(**args)
+
+    def create(self, payload):
+        params = payload.get('parameters', {})
+        sim_name = payload.get('name', 'Kubernetes Simulator')
+        jobname = params.get('jobname', 'noname')
+        adls = params.get('activeDeadlineSeconds', 3600)
+        if type(adls) is str:
+            adls = int(adls)
+        image = params.get('image')
+        name = params.get('name')
+        privileged = params.get('privileged', False)
+        uuid = params.get('uuid')
+        self.logger.info('uuid:')
```

Comment:
Please improve the logging message here. This does not need to be spread across two invocations.